static_inventories/betabarrel_cluster.yml

```diff
@@ -238,11 +281,61 @@ all:
             - name: 'umcg-vipt'
           ssh_client_configs:
             - logs_library
+    storage_server:
+      hosts:
+        bb-storage:
+          ansible_host: bb-porch-1336+bb-storage
```

Comment:
if new jumphost was addes, which has 1336 network AND that has set up /etc/hosts, then this part is not needed anymore